src/test/java/Homework18.java

```diff
@@ -1,12 +1,15 @@
+
 import org.openqa.selenium.interactions.Actions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
+
 public class Homework18 extends BaseTest {
 
     @Test
+
     public static void playSong () throws InterruptedException {
         navigateToPage();
         Thread.sleep(2000);
```

Comment:
It's a bad practice to use Thread.sleep()  - Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.